### PR TITLE
Allow popover to be dismissed

### PR DESCRIPTION
**Pull request by gcasa**

### CODE REVIEW

File: Source/NSPopover.m [hunk 1]
```diff
@@ -232,23 +232,27 @@ - (void) close
 {
   [_realPanel close];
   [_realPanel setDelegate:nil];
+  [_realPanel orderOut: nil];
```
Comment (fredkiefer) [inline Source/NSPopover.m:235]: Why is the `orderOut:` needed here and in the next method? The close Methods should already take care of that.

File: Source/NSViewController.m [hunk 2]
```diff
@@ -164,6 +186,45 @@ - (NSBundle *)nibBundle
...
+  [viewController viewDidDisappear: NO];
+}
+
+- (void) dismissController: (id)sender
+{
+  [self dismissViewController: self];
+}
+
+// NSSeguePerforming methods...
+- (void)performSegueWithIdentifier: (NSStoryboardSegueIdentifier)identifier
```
Comment (fredkiefer) [inline Source/NSViewController.m:201]: Here and in the next methods a space after the return type is missing.

Comment (gcasa) [reply Source/NSViewController.m]: Fixed, resolve if you agree.

File: Source/NSPopover.m (hunk 1 above)
Comment (gcasa) [reply Source/NSPopover.m]: This should now be addressed.

**Comment by gcasa** [comment]
I'm fixing some positioning issues with the popover now.

**Review by fredkiefer — APPROVED** [review]

**Comment by gcasa** [comment]
I am opening another PR for the positioning issue as well as the segue problems we discussed a while ago.

**Pull request merged**